sources/packages/api/src/route-controllers/institution-locations/institution-locations.controller.ts

```diff
@@ -94,6 +77,48 @@ export class InstitutionLocationsController extends BaseController {
     return createdInstitutionlocation.id;
   }
 
+  @Patch(":locationId")
+  async update(
+    @Param("locationId") locationId: number,
```

Comment:
This method needs to validate if the user updating the location has access to it.